dockerfiles/image.sh

```diff
@@ -43,8 +43,8 @@ finalize_platform_core_image() {
     local cmd=$2
     local image_name=platform-core:$MODE
     echo "--- Finalize '$image_name' image ---"
-    if [ "$ACTION" == "incr" ] && [[ $(docker images -q hpccsystems/platform-core:$MODE-incr-prev 2> /dev/null) ]]; then
-        local image_name=hpccsystems/platform-core:$MODE-incr-prev
+    if [ "$ACTION" == "incr" ] && [[ $(docker images -q prev-core:$MODE 2> /dev/null) ]]; then
+        local image_name=prev-core:$MODE
```

Comment:
"prev" for the new image name seems confusing to me.
I think it's the current incr build ? 
as per offline discussion, I think "incr-core" image name would be better.

Reply:
Renamed